repos/system_upgrade/el8toel9/actors/firewalldcheckallowzonedrifting/actor.py

```diff
@@ -0,0 +1,44 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import FirewalldGlobalConfig, FirewallsFacts
+from leapp.reporting import create_report, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class FirewalldCheckAllowZoneDrifting(Actor):
+    '''
+    This actor will check if AllowZoneDrifiting=yes in firewalld.conf. This
+    option has been removed in RHEL-9 and behavior is as if
+    AllowZoneDrifiting=no.
+    '''
```

Comment:
@erig0 pls use triple double quotes for all docstrings
* https://leapp.readthedocs.io/en/latest/python-coding-guidelines.html#write-docstrings-that-work-well-with-tools
* https://www.python.org/dev/peps/pep-0257/

Reply:
done